exercises/concept/dancing-dots/.docs/hints.md

```diff
@@ -0,0 +1,52 @@
+# Hints
+
+## General
+
+- Read about behaviours in the official [Getting Started guide][getting-started-behaviours].
+- Read about behaviours on [elixirschool.com][elixir-school-behaviours].
+- Read about behaviours in the [documentation][doc-behaviours].
```

Comment:
Thanks, but in Elixir, "behaviour" is a concept and whenever we refer to this concept, we need to use the UK spelling because it affects the code too (e.g. you need to write `@behaviour DancingDots.Animation`, not `@behavior`). Other times, when we use "behavior" in its regular life non-Elixir meaning, we use the US spelling.

There was one mistake in my PR, I fixed it: [`5827335` (#1103)](https://github.com/exercism/elixir/pull/1103/commits/5827335fb684a5c9fe7914f6f09efcc84d7a1909)